_posts/GaKay/2016-01-14-GaKays-Trinket-Programs.md

```diff
@@ -0,0 +1,13 @@
+---
+author: gao14g
+layout: post
+---
+
+This is my own personal Trinket.
+<iframe src="https://trinket.io/embed/python/e2833e5ffe" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This trinket asks for user inputs.
+<iframe src="https://trinket.io/embed/python/3770607d53" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+My Reflection:
+I thought this assignment was very fun. It was cool to see a few simple lines of code turn into a full image. For certain parts of my code, I had to use a trial-and-error method. For example, I wanted my turtle to draw a sun by first relocating to a new location. Since I wasn't sure what the exact coordinates of the location were, I had to input random coordinates and adjust them accordingly. Overall, it was an interesting exercise, and it was fun to mess around with the different functions of Trinket.
```

Comment:
Glad you enjoyed it!  Trial and error is really important.  As we go along your 'trials' will become more scientific and your 'errors' more informative.